terraform/modules/hub/files/cloud-init/prometheus.sh

```diff
@@ -68,59 +68,14 @@ run-until-success apt-get install --yes docker.io
 cat <<EOF > /etc/systemd/system/docker.service.d/override.conf
 [Service]
 ExecStart=
-ExecStart=/usr/bin/dockerd --log-driver journald --dns 10.0.0.2
+ExecStart=/usr/bin/dockerd --log-driver awslogs --log-opt awslogs-region=eu-west2 --log-opt awslogs-group ${cloudwatch_log_group} --dns 10.0.0.2
```

Comment:
```suggestion
ExecStart=/usr/bin/dockerd --log-driver awslogs --log-opt awslogs-region=eu-west-2 --log-opt awslogs-group ${cloudwatch_log_group} --dns 10.0.0.2
```
I don't know that this is right, but `eu-west2` looks wrong